src/posix/platform/netif.cpp

```diff
@@ -1055,6 +1055,39 @@ static const uint8_t *getIcmp6RaMessage(const uint8_t *data, ssize_t length)
     return ret;
 }
 
+/**
+ * Returns false if the address is a Required Anycast address.
+ *
+ */
+static bool IsRequiredAnycast(const ot::Ip6::Address *aAddress)
+{
+    uint8_t prefixLength      = aAddress->GetPrefix().kLength;
```

Comment:
`kLength` is a constant. (Defined in `ot::Ip6::NetworkPrefix`. For any address the length is the same). I'm not sure if this is correct. Is this expected? Because if the length is a constant, we don't need to get it from an address.

Reply:
ot::Ip6::Address has no function called GetLength()

Reply:
Maybe I can use the otIp6SetAddressCallback(gInstance, XXX, gInstance);